va/va.h

```diff
@@ -1130,6 +1130,50 @@ VAStatus vaDestroyContext (
     VAContextID context
 );
 
+//Multi-frame context
+typedef VAGenericID VAMFContextID;
+/**
+ * vaCreateMFContext - Create a multi-frame context
+ *  Multi-frame context allow to run tasks from different
+ *  contexts in single batch buffer for performance optimization.
```

Comment:
the underlying HW might not support batch buffer, could you use some common words?

Reply:
FIxed